aptos-move/e2e-tests/goldens/language_e2e_testsuite__tests__create_account__create_account.exp

```diff
@@ -5,18 +5,20 @@ Ok(
+                            StateKey::AccessPath { address: 0xb26af11b4f332a7794398554b6313a38c6a102c74c5ba9a12629ae3a492acb2f, path: "ResourceGroup(0x1::object::ObjectGroup)" }: Modification(020000000000000000000000000000000000000000000000000000000000000001066f626a6563740a4f626a656374436f726500590100000000000400000000000000000000000000000000000000000000000000000000000a550c180000000000000000000000000000000400b26af11b4f332a7794398554b6313a38c6a102c74c5ba9a12629ae3a492acb2f00000000000000000000000000000000000000000000000000000000000000010e66756e6769626c655f61737365740d46756e6769626c6553746f72650029000000000000000000000000000000000000000000000000000000000000000afc2fffffffffffff00, metadata:StateValueMetadata { inner: Some(StateValueMetadataInner { slot_deposit: 0, bytes_deposit: 0, creation_time_usecs: 0 }) }),
                         },
                     },
                 ),
             ),
             events: [
-                ModuleEvent { type: Struct(StructTag { address: 0000000000000000000000000000000000000000000000000000000000000001, module: Identifier("account"), name: Identifier("CoinRegister"), type_args: [] }), event_data: "f5b9d6f01a99e74c790e2f330c092fa05455a8193f1dfc1b113ecc54d067afe100000000000000000000000000000000000000000000000000000000000000010a6170746f735f636f696e094170746f73436f696e" },
-                ModuleEvent { type: Struct(StructTag { address: 0000000000000000000000000000000000000000000000000000000000000001, module: Identifier("transaction_fee"), name: Identifier("FeeStatement"), type_args: [] }), event_data: "09000000000000000500000000000000040000000000000000000000000000000000000000000000" },
+                ModuleEvent { type: Struct(StructTag { address: 0000000000000000000000000000000000000000000000000000000000000001, module: Identifier("fungible_asset"), name: Identifier("Withdraw"), type_args: [] }), event_data: "b26af11b4f332a7794398554b6313a38c6a102c74c5ba9a12629ae3a492acb2f0100000000000000" },
+                ModuleEvent { type: Struct(StructTag { address: 0000000000000000000000000000000000000000000000000000000000000001, module: Identifier("fungible_asset"), name: Identifier("Deposit"), type_args: [] }), event_data: "3f4fc7462763e539d6c50356f9c1d14ffc32b567d038da6bdfa454b8618910670100000000000000" },
+                ModuleEvent { type: Struct(StructTag { address: 0000000000000000000000000000000000000000000000000000000000000001, module: Identifier("transaction_fee"), name: Identifier("FeeStatement"), type_args: [] }), event_data: "02d000000000000004000000000000000600000000000000f8cf0000000000000000000000000000" },
             ],
-            gas_used: 9,
+            gas_used: 53250,
```

Comment:
Isn't this a huge gas change?

Is there a behavior change here?

Reply:
Yes, I changed the test since the old case does not exist anymore.